decidim-core/app/models/decidim/authorization.rb

```diff
@@ -47,6 +47,21 @@ def granted?
       !granted_at.nil?
     end
 
+    # Returns true if the authorization is renewable by the participant
+    def renewable?
+      return unless workflow_manifest
+      return unless renewable_at < Time.current
```

Comment:
This is a guard or it is part of the "method logic"? I feel that it could be part of the main condition (deleting this line):

```ruby
workflow_manifest.renewable && renewable_at < Time.current
```

Reply:
Yes 1 line makes more sense 😁